pagopa/io-react-native-integrity
# feat: [SIW-1043] Add `CBOR` decoding assertion function on iOS side

### Short description
This PR adds SwiftCBOR package to easily decoding CBOR assertion on iOS. So this PR introduces a new native function `decodeAssertion`. The output should be send to the backend.

Using CBOR on native side remove from the backend written in ts the computational cost of managed CBOR encoded data.

#### List of Changes

- Add SwiftCBOR
- Updated exmaple app
- Update backend

#### How Has This Been Tested?

- Set value in .env file under example app and backend
- Start metro with `cd example && yarn start`
- Start backend with `cd backend && yarn start`
- Using a supported iOS device open Xcode and start the example app
- Try sequentially all the CTA
- Last CTA `Verify Assertion` should return a value like this `{result: {signCount: 1}}`
